packages/devcmd-cli/src/index.ts

```diff
@@ -54,9 +54,31 @@ async function isDir(path: string): Promise<boolean> {
 async function runInDevCmdsDir(dirPath: string) {
   const [, , ...args] = process.argv;
 
-  // TODO: use spawn or so instead
-  execFileSync("node", ["-e", `require('devcmd/from-cli').run(...process.argv.slice(1))`, ...args], {
-    cwd: dirPath,
-    stdio: "inherit",
+  try {
+    await startProcess('node', ["-e", `require('devcmd/from-cli').run(...process.argv.slice(1))`, ...args], dirPath);
+  } catch (err) {
+    process.exit(err);
+  }
+}
+
+async function startProcess(command: string, args: Array<string>, dirPath: string): Promise<number> {
+  const spawnOptions: SpawnOptions = {
+    stdio: 'inherit',
```

Comment:
Do we want to show the actual error to the user? If not, this can be changed to 'ignore'